packages/@unimodules/react-native-adapter/android/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,11 @@
+<manifest package="org.unimodules.adapters.react"
+  xmlns:android="http://schemas.android.com/apk/res/android">
 
-<manifest package="org.unimodules.adapters.react">
+  <application>
+    <meta-data
+      android:name="org.unimodules.core.AppLoader#react-native-headless"
```

Comment:
Is there some convention around whether to use camelcased or snakecased property names?

Reply:
I do not know of any.